src/api/json/catalog.json

```diff
@@ -234,6 +234,26 @@
       "fileMatch": ["appsettings.StormPetrel.json"],
       "url": "https://raw.githubusercontent.com/Scandltd/storm-petrel/main/generator/assets/appsettings.StormPetrel.Schema.json"
     },
+    {
+      "name": "aqua.yaml",
+      "description": "aqua configuration file",
```

Comment:
Again, similarly in lowercase per upstream conventions as in https://github.com/SchemaStore/schemastore/pull/4587

Reply:
Sounds good